src/js/overlay.js

```diff
@@ -22,6 +22,14 @@ const checkOptions = function(opts) {
 		throw new Error('"o-overlay error": Compact overlays can\'t have a shaded header');
 	}
 
+	// Map old names for options
+	if (opts.parentNode) {
+		opts.parentnode = opts.parentNode;
+	}
+	if (opts.noFocus) {
+		opts.nofocus = opts.noFocus;
+	}
```

Comment:
We recently added a [deprecation guide](http://origami.ft.com/docs/component-spec/modules/#deprecation-of-a-module-feature) that covers what to do in situations like this. What you've done is great but it'd be useful to include a `console.warn` in each of these conditionals explaining that they're deprecated.

Reply:
Thanks!  Could you add a `type: breaking` label to the project so that I can create the new issue?  (I don't seem to be able to create the label myself)